plugins/inputs/mysql/mysql.go

```diff
@@ -24,6 +30,10 @@ var sampleConfig = `
   #
   # If no servers are specified, then localhost is used as the host.
   servers = ["tcp(127.0.0.1:3306)/"]
+  PerfEventsStatementsDigestTextLimit = 120
```

Comment:
Also, for each of these, can you explain the use-case of someone wanting to adjust these? Seems like a few of these could just be hardcoded constants within the config file, but I'm not as familiar with these metrics.